.gitignore

```diff
@@ -38,6 +38,7 @@ docs/_build
 env
 env*
 venv
+.venv
```

Comment:
What's wrong with `venv` ;)

Reply:
Ah good point, in the readme for the virtual env install it's `venv`. I would argue to make a change in a quick PR to `.venv` as this is what we use in our Python Flask work and exploratory work. 

It's very a minor thing, but we should pick one or another :)

Reply:
I'm easy. But yes, it's right to take it out of this PR